.bazelrc

```diff
@@ -54,9 +54,7 @@ build --nolegacy_external_runfiles
 ## Bzlmod
 ###############################################################################
 
-# TODO: migrate all dependencies from WORKSPACE to MODULE.bazel
-# https://github.com/bazelbuild/rules_rust/issues/2181
-common --noenable_bzlmod
+common --experimental_enable_bzlmod
```

Comment:
Can we put this in its own PR and get it submitted first? Speaking from experience when migrating our product to bzlmod, I don't have faith that this won't break anything, and I'd like to be able to revert this change individually..

Reply:
Yeah. That was an accident.